pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -362,6 +362,10 @@ stdenv.mkDerivation {
     ''
 
     + ''
+      # There are a few tools (to name one libstdcxx5) which do not work
+      # well with multi line flags, so make the flags single line again
```

Comment:
Can we move the comment outside of the string so that it doesn't effect the derivation hash?

Reply:
Done :)